wiki/Game_modifier/Nightcore/ko.md

```diff
@@ -0,0 +1,48 @@
+---
+태그:
+  - nightcore
+  - NC
+  - mod
+  - game modifier
+---
+
+# 나이트코어 (모드)
+
+![NC mod icon](/wiki/shared/mods/NC.png "나이트코어 (NC) 모드 아이콘")
+
+*모든 모드의 목록을 확인하려면 [여기](/wiki/Game_modifier)를 눌러주세요.*\
+*[나이트코어 (장르)](https://en.wikipedia.org/wiki/Nightcore "위키피디아")와 혼동하지 말아주세요.*\
+*비슷한 모드: [더블 타임 (모드)](/wiki/Game_modifier/Double_Time)*
+
+## 나이트코어 모드
+
+- 축약형: NC
+- 종류: 난이도 상승 모드
+- 점수 배수:
+  - ![][o!s]: 1.12x
+  - ![][o!t]: 1.12x
+  - ![][o!c]: 1.06x
+  - ![][o!m]: 1.00x
+- 기본 단축키: `D` `D` 또는 `Shift`+`D`
+- 설명: `꺄아아아아아아아아!!`
+- NC를 사용할 수 있는 osu!의 장르: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+- 같은 계열의 모드: [더블 타임](/wiki/Game_modifier/Double_Time)
+
+## 상세 설명
+
+[**나이트코어**](/wiki/Game_modifier)는 같은 계열의 모드인 더블 타임과 정확히 동일하게 작동하는 모드입니다.
+하지만 이 모드는 음의 높이를 높이고 드럼 소리를 백그라운드에 추가하여 전체 4박자에서 첫 번째와 세 번째 박자는 베이스 드럼으로, 두 번째와 네 번째 박자는 박수를 치는 것을 반복합니다.
+
+더블 타임 모드가 활성화 되었을 때와 동일한 [점수 배수](/wiki/Game_modifier/Score_multiplier)를 유지하며, 비트맵 자체는 더블 타임 모드의 효과를 넘어서는 영향을 받지 않습니다.
+
+이 모드는 모든 osu! 게임 장르에서 동일한 효과를 가집니다.
+
+## 그 외
+
+- 나이트코어 모드는 본래 [만우절 장난](https://osu.ppy.sh/community/forums/topics/49733)이었지만, 실제로 플레이가 가능한 모드로 추가 되었습니다.
+- 나이트코어 모드는 음악을 편집하거나 생성할 때 대략 10-30%의 속도를 높이고 더 높은 음조의 목소리를 갖는 "나이트코어" 장르에서 이름을 따왔습니다. 이런 종류의 음악의 타당성과 역사는 논쟁의 여지가 있긴 합니다.
```

Comment:
```suggestion
- 나이트코어 모드는 본래 [만우절 장난](https://osu.ppy.sh/community/forums/topics/49733)이었지만, 추후에 실제로 플레이가 가능한 모드로 추가 되었습니다.
- 나이트코어 모드는 음악을 편집하거나 생성할 때 대략 10-30%의 속도를 높이고 더 높은 음조의 목소리를 갖는 "나이트코어" 장르에서 이름을 따왔습니다. 단, 이런 종류의 음악의 타당성과 역사는 논쟁의 여지가 있긴 합니다.
```
Since nightcore as an April Fools and nightcore as a mod has some time between it, I suggest adding the word 추후에 to indicate that the introduction of a nightcore as a mod was made after some time has past after the initial April Fools joke.

Also on last sentence, it is first introduced as a genre but adds a statement that approval of these genres is disputed. So I suggest adding 단, for more natural transition between statements.